coldfront/overlays/ocp-staging/postgrescluster.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: coldfront-database
+  namespace: coldfront
+spec:
+  image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.4-0
+  postgresVersion: 14
+  instances:
+    - name: instance1
+      dataVolumeClaimSpec:
+        accessModes:
+          - "ReadWriteOnce"
+        resources:
+          requests:
+            storage: 1Gi
+  backups:
+    pgbackrest:
+      image: registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi8-2.38-2
```

Comment:
Check with @naved001 : do we need to provide an image here, or will the operator Do The Right Thing without this?

Reply:
It should be left out and the operator will do the right thing, see: https://github.com/CCI-MOC/moc-apps/pull/178

Reply:
Awesome, thanks, will update. The example included it so I left it there.